### PR TITLE
[Camera] Add new APIs for manual focus level, product id and vendor id

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Add new APIs for manual focus level, product id and vendor id

### API Changes ###

 - TCSACR-545

### CODE REVIEW

File: src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs [hunk 1]
```diff
@@ -113,6 +113,36 @@ public IEnumerable<CameraDeviceInformation> SupportedDevices
             }
         }
 
+        /// <summary>
+        /// Gets the product id for given <param name="device"/>.
```
Comment (wiertel) [inline src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs:117]: please write "ID" in the text, same for line 132,
lines 126 and 141 can stay unchanged

File: src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs [hunk 2]
```diff
@@ -113,6 +113,36 @@ public IEnumerable<CameraDeviceInformation> SupportedDevices
...
+        /// </summary>
+        /// <exception cref="ArgumentException">Invalid enumeration.</exception>
+        /// <exception cref="ObjectDisposedException">The CameraDeviceManager already has been disposed.</exception>
+        /// <since_tizen> 11 </since_tizen>
+        public ushort GetProductId(CameraDevice device)
```
Comment (wiertel) [inline src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs:122]: please consider changing `ushort` -> `UInt16`, those are exactly the same type (one is an alias for another) so nothing needs to change apart form return type, but would be more explicit which is better for public API
same in line 137

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs:122]: ushort is exactly same with UInt16. I just want to keep this like other alias types.

File: src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs]: Fixed

File: src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs (hunk 2 above)
Comment (wiertel) [reply src/Tizen.Multimedia.Camera/Camera/CameraDeviceManager.cs:122]: ok, `ushort` is not used very often, but both styles are used across TizenFX, so I will not object

**Review by wiertel — APPROVED** [review]

**Pull request merged**